tideways/usr/local/share/env/40-stack

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export TIDEWAYS_HOSTNAME=${TIDEWAYS_HOSTNAME:-tideways-daemon}
+export TIDEWAYS_ENVIRONMENT=${TIDEWAYS_ENVIRONMENT:-production}
```

Comment:
Isn't it `default` by default? 🤔

Reply:
https://tideways.io/profiler/article/42-environments says `production`